src/fu-device.c

```diff
@@ -1190,12 +1189,6 @@ fu_device_set_version (FuDevice *self, const gchar *version)
 	} else {
 		version_safe = g_strdup (version);
 	}
-
-	/* try to autodetect the version-format */
-	if (version_format == FWUPD_VERSION_FORMAT_UNKNOWN) {
-		version_format = fu_common_version_guess_format (version_safe);
-		fu_device_set_version_format (self, version_format);
-	}
```

Comment:
Oh sorry, I missed this in a few earlier comments.  Should this accompanying function (`fu_common_version_guess_format `) also be nuked then too?

Reply:
I'm reusing it a follow up commit; pretend it has disappeared. :)